doc/examples/gestures/lib/main_multitap_advanced.dart

```diff
@@ -49,37 +50,31 @@ class MyGame extends BaseGame
   }
 
   @override
-  void onReceiveDrag(DragEvent event) {
-    onPanStart(event.initialPosition);
-
-    event
-      ..onUpdate = onPanUpdate
-      ..onEnd = onPanEnd
-      ..onCancel = onPanCancel;
-  }
-
-  void onPanCancel() {
+  void onDragCanceled(int pointerId) {
     _end = null;
     _start = null;
     _panRect = null;
   }
 
-  void onPanStart(Offset position) {
+  @override
+  void onDragStarted(int pointerId, Vector2 initialPosition) {
     _end = null;
-    _start = position;
+    _start = initialPosition;
   }
 
-  void onPanUpdate(DragUpdateDetails details) {
-    _end = details.localPosition;
+  @override
+  void onDragUpdated(int pointerId, DragUpdateDetails details) {
+    _end = details.localPosition.toVector2();
   }
 
-  void onPanEnd(DragEndDetails details) {
+  @override
+  void onDragEnded(int pointerId, DragEndDetails details) {
     if (_start != null && _end != null) {
       _panRect = Rect.fromLTRB(
-        _start.dx,
-        _start.dy,
-        _end.dx,
-        _end.dy,
+        _start.x,
```

Comment:
Could be `_start.toPositionedRect(_end-_start)` too, not sure what is the easiest to read.

Reply:
much better, I will fix on the other repo